### PR TITLE
add prefixes to redis session config

### PR DESCRIPTION
**Pull request by buddhaCode**
We had some issues with this. Just to let others know.

### CODE REVIEW

File: source/sysadmins-guide/sessions/index.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ have a stand alone memcache instance in place:
 ### Redis
 Redis is a popular key/value storage, that easily can be clustered for redundancy. It does not support session locking,
 however.
+When using a single redis instance with multiple shopware installations (e.g. for stating environments) it would be wise to use seperate prefixes for each shopware installation. Otherwise, your session keys could re-used between your installations and race conditions or strange session-related behavoir may occur. A prefix can be confiured using the conncection uri. Please consult the offical [phpredis documentation](https://github.com/phpredis/phpredis#php-session-handler).
```
Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Typo: stating -> staging

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Typo: seperate -> separate

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Grammar: 

```diff
- your session keys could re-used
+ your session keys could be re-used
```

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Typo: behavoir -> behavior

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Typo: offical -> official

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Typo:  confiured  -> configured

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Typo: conncection -> connection

Comment (bcremer) [inline source/sysadmins-guide/sessions/index.md]: Shopware as in the Product should be written uppercase "Shopware".
Shopware as in the company should be written lowercase "shopware AG".

**Comment by buddhaCode** [comment]
how can one possibly make so much mistakes in just sour sentences.  😢

**Comment by bcremer** [comment]
No worries. Thanks for your contribution 👍

**Comment by mitelg** [comment]
thanks @buddhaCode :+1: :tada: 

also thanks @bcremer for the review :smile: :+1:

**Pull request merged**